pkg/volume/volume.go

```diff
@@ -132,3 +133,22 @@ func RenameDirectory(oldPath, newName string) (string, error) {
 	}
 	return newPath, nil
 }
+
+// Return the wrapper volume spec of specific volume plugin
+func GetWrappedVolumeSpec(volName string, pluginName string) *Spec {
+	// The name of wrapper volume is set to "wrapped_{wrapped_volume_name}"
+	wrapperVolumeName := fmt.Sprintf("wrapped_%v", volName)
+
+	var volumeSpec *Spec
+	switch pluginName {
+	case "kubernetes.io/downward-api", "kubernetes.io/secret":
```

Comment:
So does it do harm if we make a copy internally? I think changing type of `spec` will affect behavior of other irrelevant plugins.